src/pages/edge/adobe-journey-optimizer/release-notes.md

```diff
@@ -9,6 +9,32 @@ keywords:
 
 # Release notes
 
+## June 9, 2025
+
+### React Native Messaging 7.1.0
+
+## Add support for tracking interactions with content cards
+
+### trackContentCardDisplay
+
+* Tracks a Display interaction with the given ContentCard
+
+#### Syntax
+
+```
+Messaging.trackContentCardDisplay(proposition, contentCard);
+```
+
+### trackContentCardInteraction
+
+* Tracks a Click interaction with the given ContentCard
+
+#### Syntax
+
+```
+Messaging.trackContentCardInteraction(proposition, contentCard);
+```
+
```

Comment:
Can we make the ReactNative release notes more concise similar to the other platforms?
`Added interaction tracking support for content cards`

I am not sure if there has been a precedent of adding syntax and api in separate sections.